tests/ui/resolve/issue-109153.rs

```diff
@@ -0,0 +1,16 @@
+// check-pass
```

Comment:
Is it necessary to add some runtime check?

Reply:
No